### ISSUE TITLE
Android-Kotlin : 09.3 Spinner in Android (Kotlin)

### ISSUE THREAD

**Issue by sitamadex11** [girlscript/winter-of-contributing#1393]
<hr>

## Description 📜

* Give a step by step tutorial of Spinner in Android (Kotlin)
*  Required Documentation or Video or Audio.

<hr>

## Domain of Contribution 📊

- [x] Android Dev (Flutter, Java, Kotlin)

<hr>

<hr>

## References :
> https://www.geeksforgeeks.org/spinner-in-android-using-java-with-example/#:~:text=Android%20Spinner%20is%20a%20view,when%20we%20click%20on%20it.

<hr>

## Please Read it Carefully Before Contributing
* Changes should be made in **Android_Development_With_Kotlin/** directory under **Android_Development_With_Kotlin** branch.
* Inside **Android_Development_With_Kotlin/** directory creates a directory and names it according to the topic ( for this issue **09. Dynamic User Interface**)
* Inside **09. Dynamic User Interface/** directory add your task and make changes accordingly.
* You have to push your PR to the **Android_Development_With_Kotlin** branch only.
* You can take help to the references but don't simply try to copy it. Just explain in your own word.
* This issue is for **GWOC'21** contributors only.
* You need to push in a single commit. If there exists more than one commit then you need to **squash** your commits into one commit.
* Will assign you on a first come first serve basis.
* Please try to complete your assigned task within 7 days. If you need some more time then, please inform mentors and supervisors.
* If you have two previously assigned issues on **Android-Kotlin** pending then please first complete them.
> Supervisor @rahulsain

## Happy Contributing. Best of Luck.💖

**Comment by Tanushree-coder** [comment]
I want to work on this issue in the documentation section.
Tanushree here ! (Batch-1, Android-Dev-Kotlin)
@sitamadex11

**Comment by sitamadex11** [comment]
yeah you can start working. And you can assign yourself also if you are commenting first by typing simply "/assign".

**Comment by Tanushree-coder** [comment]
> yeah you can start working. And you can assign yourself also if you are commenting first by typing simply "/assign".

okay sure sir !
